services/galley/src/Galley/API/Update.hs

```diff
@@ -1000,7 +1001,7 @@ withValidOtrRecipients protectee clt cnv rcps val now go = do
     Data.deleteConversation cnv
     throwM convNotFound
   -- FUTUREWORK(federation): also handle remote members
-  localMembers <- Data.members cnv
+  (FutureWork @'LegalholdPlusFederationNotImplemented -> _remoteMembers, localMembers) <- (undefined,) <$> Data.members cnv
```

Comment:
Does this have to be `undefined`?

Reply:
Do you prefer `()`?  I think I prefer `undefined` because it makes sure this makes it more clear it shouldn't be used anywhere.  Perhaps there is a better idiom yet to be found though; I tried something diferent [here](https://github.com/wireapp/wire-server/blob/a843ef28039ff9cb85d2ef57216076114547f488/services/brig/src/Brig/Provider/API.hs#L865)